sdk/tables/Azure.Data.Tables/tests/TableClientQueryableLiveTests.cs

```diff
@@ -414,83 +414,83 @@ public async Task TableQueryableWithDictionaryTypeOnSupportedTypes()
             }
 
             // 1. Filter on String
-            var results = await client.QueryAsync(ent => (ent["String"] as string).CompareTo(thirdEntity.String) >= 0).ToEnumerableAsync().ConfigureAwait(false);
+            var results = await client.QueryAsync<TableEntity>(ent => (ent["String"] as string).CompareTo(thirdEntity.String) >= 0).ToEnumerableAsync().ConfigureAwait(false);
```

Comment:
these should all change to use the strongly typed value getters e.g.
```c#
ent => (ent.GetString("String"))
```

Reply:
Should we do both to make sure there are no unintentional differences?

Reply:
> Should we do both to make sure there are no unintentional differences?

I'll keep both just in case. I caught an error earlier with the `TableEntity`'s indexer not working with the LINQ expression parser(?), so it might matter later.